test_rclcpp/test/test_timer.cpp

```diff
@@ -59,7 +59,7 @@ TEST(CLASSNAME(test_time, RMW_IMPLEMENTATION), timer_fire_regularly) {
     printf("spin_node_some() for 4s\n");
     while (std::chrono::steady_clock::now() < start + 4.5 * period) {
       executor.spin_node_some(node);
-      std::this_thread::sleep_for(period / 25);
```

Comment:
Got it, I didn't see that you reduced the period.

Then you may want to change the print message `spin_node_some() for 4s` in line 59 to `400ms`.